### DISCUSSION TITLE
Make a single photo in a row take the full width of the Photo Album component

### DISCUSSION THREAD

**Discussion by faustonunez** [igordanchenko/react-photo-album#162] [Q&A] [answered]
Hi all! I'm wondering if there is a way through the Photo Album properties to target a photo or multiple photos in different rows to take the full width of a row. The idea is that I can still have the targetRowHeight that programmatically decides how many photos show on a row based on height, but I can make exceptions for certain photos and make them instead take the full width of a given row. Thanks!

**Comment by igordanchenko** [comment, marked as answer]
Hi there. I'm afraid this is not something supported by this library. All supported properties are covered in the [documentation](https://react-photo-album.com/documentation). However, if you know in advance which photo you want to take the whole row, maybe you can split your photos in several batches and stack several photo albums on top of each other?

```jsx
<PhotoAlbum ... />
<img ... />
<PhotoAlbum ... />
```

**Reply by faustonunez** [reply]
Thank you so much for the quick response! I had already taken this approach, but I was hoping there was a more streamlined way of doing it instead of breaking things up into batches, which require more maintenance. Also, I'm using Yet Another React Lightbox, and when I break photos into multiple Albums, it seems like I need a Lightbox component for each album instead of just one, which also makes all the albums in the Lightbox not connected.

**Reply by igordanchenko** [reply]
> Also, I'm using Yet Another React Lightbox, and when I break photos into multiple Albums, it seems like I need a Lightbox component for each album instead of just one, which also makes all the albums in the Lightbox not connected.

I'm pretty sure you can use a single instance of the lightbox in this case.

**Reply by faustonunez** [reply]
If you have an example of how to have two Photo Albums link to one Lightbox, I would greatly appreciate it. For more context, I'm using a similar structure as [this example  ](https://stackblitz.com/github/igordanchenko/react-photo-album/tree/main/examples/lightbox?description=react-photo-album%20with%20lightbox&file=src%2FApp.tsx&title=react-photo-album-lightbox). I have a photos.ts with all my images and then a component that renders the PhotoAlbum and Lightbox.

**Reply by igordanchenko** [reply]
Sure, here is one way to implement it - https://stackblitz.com/edit/react-photo-album-162?file=src%2FApp.tsx

**Reply by faustonunez** [reply]
This works perfectly!  photos={photos.slice(0, 6)} is doing the trick. I greatly appreciated!
